service/src/main/java/uk/gov/hmcts/reform/fpl/service/email/content/base/AbstractEmailContentProvider.java

```diff
@@ -54,4 +58,22 @@ protected String getDocumentUrl(DocumentReference document) {
         }
         return "";
     }
+
+    protected String getJudgeName(AbstractJudge judgeAndLegalAdvisor) {
```

Comment:
@Jamesnelson117 we can use this in your pr if you want https://github.com/hmcts/fpl-ccd-configuration/pull/2109